jobserver/views/jobs.py

```diff
@@ -64,17 +64,24 @@ def get(self, request, *args, **kwargs):
         # have more than one backend-specific error code it makes sense to keep
         # things together.
         log_path = ""
+        log_path_url = ""
         if job.job_request.backend.slug == "tpp" and job.status_code == "nonzero_exit":
             log_path = os.path.join(
                 job.job_request.backend.parent_directory,
                 job.job_request.workspace.name,
                 "metadata",
                 f"{job.action}.log",
             )
+            log_path_url = os.path.join(
+                job.job_request.workspace.get_files_url(),
+                "metadata",
+                f"{job.action}.log",
+            )
```

Comment:
While `os.path.join` will work in most cases I'd rather use `furl` here since we're constructing a URL.

Reply:
That's a fair point. Is it the correct URL though?

Reply:
Updated to use furl